File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Modifiers/AccessModifiers.cs
#region Class

namespace Class
{
    public class PublicType
    {
    }

    internal class InternalType
    {
    }

    public partial class ParentType
    {
        private class PrivateType
        {
        }

        protected class ProtectedType
        {
        }

        protected internal class ProtectedInternalType
        {
        }

        internal protected class InternalProtectedType
        {
        }
    }
}

#endregion

#region Struct

namespace Struct
{
    public struct PublicType
    {
    }

    internal struct InternalType
    {
    }

    public partial struct ParentType
    {
        private struct PrivateType
        {
        }

        protected struct ProtectedType
        {
        }

        protected internal struct ProtectedInternalType
        {
        }

        internal protected struct InternalProtectedType
        {
        }
    }
}

#endregion

#region Interface

namespace Interface
{
    public interface PublicType
    {
    }

    internal interface InternalType
    {
    }

    public partial interface ParentType
    {
        private interface PrivateType
        {
        }

        protected interface ProtectedType
        {
        }

        protected internal interface ProtectedInternalType
        {
        }

        internal protected interface InternalProtectedType
        {
        }
    }
}

#endregion

#region Enum

namespace Enum
{
    public enum PublicType
    {
    }

    internal enum InternalType
    {
    }

    public partial class ParentType
    {
        private enum PrivateType
        {
        }

        protected enum ProtectedType
        {
        }

        protected internal enum ProtectedInternalType
        {
        }

        internal protected enum InternalProtectedType
        {
        }
    }
}

#endregion

#region Delegate

namespace Delegate
{
    public delegate void PublicType();

    internal delegate void InternalType();

    public partial class ParentType
    {
        private delegate void PrivateType();

        protected delegate void ProtectedType();

        protected internal delegate void ProtectedInternalType();

        internal protected delegate void InternalProtectedType();
    }
}

#endregion

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Modifiers/AccessModifiers.vb
#Region "Class"

Namespace Class
    Class NoAccessModifierType
    End Class

    Public Class PublicType
    End Class

    Friend Class InternalType
    End Class

    Partial Public Class ParentType
        Private Class PrivateType
        End Class

        Protected Class ProtectedType
        End Class

        Protected Friend Class ProtectedInternalType
        End Class

        Protected Friend Class InternalProtectedType
        End Class
    End Class
End Namespace

#End Region

#Region "Struct"

Namespace Struct
    Structure NoAccessModifierType
    End Structure

    Public Structure PublicType
    End Structure

    Friend Structure InternalType
    End Structure

    Partial Public Structure ParentType
        Private Structure PrivateType
        End Structure

        Protected Structure ProtectedType
        End Structure

        Protected Friend Structure ProtectedInternalType
        End Structure

        Protected Friend Structure InternalProtectedType
        End Structure
    End Structure
End Namespace

#End Region

#Region "Interface"

Namespace Interface
    Interface NoAccessModifierType
    End Interface

    Public Interface PublicType
    End Interface

    Friend Interface InternalType
    End Interface

    Partial Public Interface ParentType
        Private Interface PrivateType
        End Interface

        Protected Interface ProtectedType
        End Interface

        Protected Friend Interface ProtectedInternalType
        End Interface

        Protected Friend Interface InternalProtectedType
        End Interface
    End Interface
End Namespace

#End Region

#Region "Enum"

Namespace Enum
    Enum NoAccessModifierType
    End Enum

    Public Enum PublicType
    End Enum

    Friend Enum InternalType
    End Enum

    Partial Public Class ParentType
        Private Enum PrivateType
        End Enum

        Protected Enum ProtectedType
        End Enum

        Protected Friend Enum ProtectedInternalType
        End Enum

        Protected Friend Enum InternalProtectedType
        End Enum
    End Class
End Namespace

#End Region

#Region "Delegate"

Namespace Delegate
    Delegate Sub NoAccessModifierType()

    Public Delegate Sub PublicType()

    Friend Delegate Sub InternalType()

    Partial Public Class ParentType
        Private Delegate Sub PrivateType()

        Protected Delegate Sub ProtectedType()

        Protected Friend Delegate Sub ProtectedInternalType()

        Protected Friend Delegate Sub InternalProtectedType()
    End Class
End Namespace

#End Region